DuckDuckGo/UserText.swift

```diff
@@ -543,6 +543,16 @@ In addition to the details entered into this form, your app issue report will co
 """, comment: "Breakage report footer explaining what is collected in the breakage report")
     public static let appTPReportSubmit = NSLocalizedString("appTP.report.submit", value: "Submit", comment: "Breakage report submit button")
     
+    public static let appTPHomeBlockedPrefix = NSLocalizedString("appTP.home.blockedPrefix", value: "App Tracking Protection blocked ", comment: "Prefix of string 'App Tracking Protection blocked x tracking attempts today' (note the trailing space)")
+    public static let appTPHomeBlockedSuffix = NSLocalizedString("appTP.home.blockedSuffix", value: " in your apps today.", comment: "Prefix of string 'App Tracking Protection blocked x tracking attempts today' (note the leading space)")
+    public static func appTPHomeBlockedCount(countString: String) -> String {
+        let message = NSLocalizedString("appTP.home.blockedCount", value: "%@ tracking attempts", comment: "Count part of string 'App Tracking Protection blocked x tracking attempts today'")
```

Comment:
We should make this a [part of the stringsdict](https://github.com/duckduckgo/iOS/blob/develop/DuckDuckGo/en.lproj/Localizable.stringsdict) and pluralize it, since right now the UI says "blocked 1 tracking attempts".